test/Concurrency/transfernonsendable_functionsubtyping.swift

```diff
@@ -0,0 +1,111 @@
+// RUN: %target-typecheck-verify-swift -swift-version 6
+
+// READ THIS! This file only contains tests that validate that the relevant
+// function subtyping rules for sending work. Please do not put other tests in
+// the file!
+
+// REQUIRES: concurrency
+// REQUIRES: asserts
+
+////////////////////////
+// MARK: Declarations //
+////////////////////////
+
+class NonSendableKlass {}
+
+protocol ProtocolWithSendingReqs {
+  func sendingResult() -> sending NonSendableKlass // expected-note {{}}
```

Comment:
what is expected note {{}}  -- "any note"? First time seeing this tbh; worth spelling it out?

Reply:
The reason I did this is that the message here isn't important to what I am testing but it is still a diagnostic.